ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/command/custom/CustomCommandType.java

```diff
@@ -8,7 +8,7 @@
  * Contributors:
  *   Codenvy, S.A. - initial API and implementation
  *******************************************************************************/
-package org.eclipse.che.ide.extension.machine.client.command.custom;
+package org.eclipse.che.ide.command.custom;
 
 import com.google.inject.Inject;
```

Comment:
Please, move all classes for _custom command type_: `CustomCommandType`, presenter, view and tests into _org.eclipse.che.ide.command.**type**.custom_ package